### DISCUSSION TITLE
Links to lines don't immediately color

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#46156] [Mobile] [answered]
GitHub Mobile beta
1. Load https://github.com/grafana/grafana/issues/47835
    ![Screenshot_20230202-103042.png](https://user-images.githubusercontent.com/2119212/216369293-10d20914-2efa-4d7e-9fa7-3ecedcc94407.png)

2. Click my link to a line number range
    https://github.com/grafana/grafana/blob/38809d73c246a75d0892ee9754c5737b38ac67a4/pkg/cmd/grafana-cli/commands/commands.go#L91-L98
3. The file loads but to the top, so I scroll down to the right place
     See ![Screenshot_20230202-103053.png](https://user-images.githubusercontent.com/2119212/216368762-39e01752-593c-40c1-99a4-ee0187d9e499.png)

*Eventually* this might happen:
![Screenshot_20230202-103108.png](https://user-images.githubusercontent.com/2119212/216368839-cb46b7fd-3708-4837-9d08-584f8f629dfa.png)

But I tried twice and it only happened the second time and still took so long that I was willing to file this ticket.

**Comment by alcere** [comment, marked as answer]
@jsoref thanks for reporting this issue. I can reproduce it, I will create a bug, and the team will work on it as soon as possible.
